libnopegl/src/math_utils.c

```diff
@@ -118,6 +118,12 @@ void ngli_vec4_lerp(float *dst, const float *v1, const float *v2, float c)
     dst[3] = NGLI_MIX_F32(a[3], b[3], c);
 }
 
+void ngli_vec4_perspective_div(float *dst, const float *v)
+{
+    const NGLI_ALIGNED_VEC(r) = NGLI_VEC4_SCALE(v, v[3]);
```

Comment:
I made the mistake but wasn't it supposed to be `1.f/v[3]`?

Reply:
Yep, fixed.